roles/ha_apps/etcd/tasks/check.yml

```diff
@@ -0,0 +1,29 @@
+- name: Wait for ETCD-0 pod to run
```

Comment:
Instead of checking if all the pods are in running state individually, it would be a good idea to wait until the condition `statefulset.status.readyReplicas == 3` is achieved. 

Etcd takes a little bit of time to achieve quorum even after pods get into `Running` state, hence it is not a good indicator if etcd has become available. However, the `readyReplicas` will be set to 3 only after `readiness` probe on all three pods are successful, which means once the condition is achieved, etcd is immediately available to be used.